contents/handbook/growth/billing.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Billing
+sidebar: Handbook
+showTitle: true
+---
+
+## Setting up billing
```

Comment:
I think it would be really helpful to outline explicitly for each [pricing scheme](https://posthog.com/pricing) how payment is handled (maybe a table?)

My assumption is that we do manual billing for Ultimate, Enterprise and Supported Enterprise from your instructions - is that right?

Reply:
Yes exactly, the manual billing applies to any non-cloud paid plans, or as of now if a user in a cloud plan wishes to switch plans.